R/param-estimates.R

```diff
@@ -43,17 +43,16 @@ param_estimates <- function(.summary) {
 #' @importFrom stringr str_detect
 #' @export
 param_estimates.bbi_nonmem_summary <- function(.summary) {
-  num_methods <- length(.summary[[SUMMARY_PARAM_DATA]])
```

Comment:
there were two problems, only [one of which](https://github.com/metrumresearchgroup/bbr/issues/330#issue-813667615) was diagnosed in the original issue. 

The other is this line. Specifically for `METHOD=CHAIN` there is nothing in `.summary[[SUMMARY_PARAM_DATA]]` so `num_methods` was coming up with one method _less_ than it should've had. This was causing problems further down, namely that when we thought we were checking the last estimation method we were actually checking the _second to last_. The change on lines 55-56 is also to account for this.